pkgs/servers/keycloak/scim-keycloak-user-storage-spi/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, maven
+, javaPackages
+}:
+
+javaPackages.mavenfod rec {
+  pname = "scim-keycloak-user-storage-spi";
+  version = "beta";
```

Comment:
Versions are always a version string or `unstable-yyyy-mm-dd` (date when the source was produced).